WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -183,6 +188,25 @@ public static void setStatsTimeframe(StatsTimeframe timeframe) {
         }
     }
 
+    public static CommentStatus getCommentsStatusFilter() {
+        int idx = getInt(DeletablePrefKey.COMMENTS_STATUS_TYPE_INDEX);
+        CommentStatus[] commentStatusValues = CommentStatus.values();
+        if (commentStatusValues.length < idx) {
+            return commentStatusValues[0];
+        } else {
+            return commentStatusValues[idx];
+        }
+    }
+    public static void setCommentsStatusFilter(CommentStatus commentstatus) {
```

Comment:
Retaining the current filter is a great touch. Do you think it should be reset when the user switches sites?